javascript/dwa-starter-vue/tests/App.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from 'vitest'
+
+import { mount } from '@vue/test-utils'
+import HelloWorld from '../src/App.vue'
+
+describe('HelloWorld', () => {
+  it('renders properly', () => {
+    const wrapper = mount(HelloWorld)
+    expect(wrapper.text()).toContain('Hello World')
+  })
+})
```

Comment:
@finn-tbd, @leordev hey yall! the contributor added a test as we asked. Am I okay to approve and merge their PR or is there anything else?

Reply:
@Johnnyevans32 thanks for the contribution! I'm excited to see this coming alive! 

Can you please add a [.tbd-example.json](https://github.com/TBD54566975/tbd-examples/blob/main/javascript/dinger/.tbd-example.json) file just so our CI can pick your tests? I think the file is pretty simple, it just need to know the commands to be run to install the test dependencies and then the command to run all the tests. 

(Some examples in this search: https://github.com/search?q=repo%3ATBD54566975%2Ftbd-examples%20.tbd-example.json&type=code)

Reply:
on it!

Reply:
you can review this update now

Reply:
@leordev